lib/widgets/main_app_bar.dart

```diff
@@ -1,161 +1,150 @@
-                      child: Text(
-                        "UPDATE",
-                        style: TextStyle(fontSize: 16.0),
-                      ),
-                    )
-                  ],
-                ),
-              ),
-            ),
-          ],
-        ),
-      ),
-      'custom2': CustomOverlayScreen(
-        backgroundColor: Colors.yellow[400],
-        content: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Icon(
-              Icons.check_circle_outline,
-              color: Colors.green,
-              size: 80.0,
-            ),
-            Padding(
-                padding: EdgeInsets.only(top: 20.0),
-                child: Text("Let's start!"))
-          ],
-        ),
-      ),
-    });
-    return Container(
-      padding: const EdgeInsets.symmetric(
-        vertical: 10.0,
-        horizontal: 10.0,
-      ),
-      color: Theme.of(context).primaryColor,
-      child: Row(
-        children: [
-          Expanded(
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: [
-                // Readability Settings
-                GestureDetector(
-                  onTap: () async {
-                    OverlayScreen().show(
-                      context,
-                      identifier: 'display',
-                    );
-                  },
-                  child: Icon(
-                    Icons.apps,
-                    color: Colors.white,
-                    size: 25.0,
-                  ),
-                ),
-                // Board Name
-                Text(
-                  'Board Name',
-                  style: TextStyle(color: Colors.white, fontSize: 20.0),
-                ),
-                // Lock
-                GestureDetector(
-                  onTap: () => print('Lock'), // insert new screen here
-                  child: Icon(
-                    Icons.lock,
-                    color: Colors.white,
-                    size: 25.0,
-                  ),
-                ),
-              ],
-            ),
-          ),
-        ],
-      ),
-    );
-  }
-}
+import 'package:flutter/material.dart';
+// import 'widgets.dart';
+import 'package:overlay_screen/overlay_screen.dart';
+
+class MainAppBar extends StatelessWidget {
+  final double scrollOffset;
+
+  const MainAppBar({
+    Key key,
+    this.scrollOffset = 0,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    OverlayScreen().saveScreens({
+      'display': CustomOverlayScreen(
+        backgroundColor: Colors.transparent,
+        content: Column(
+          mainAxisAlignment: MainAxisAlignment.end,
+          children: <Widget>[
+            SizedBox(
+              height: 200,
+              width: double.infinity,
+              child: DecoratedBox(
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                ),

```

Comment:
I see here you used a `DecoratedBox` to make this component white - can you adjust the color of this widget on the background of `Card` instead? That might simplify some of the design here - I'm not all too sure about that though 😅 .